POS_Image-RPi-Bootstrap/rpi-bootstrap-7.0.0/config.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="6.1" name="POS_Image_RPi-Bootstrap">
```

Comment:
Can we use up to date schema version for kiwi  (7.3) and be compliant with it?

Reply:
This is copied from JeOS image.
IMHO we should use the same  schema version in all images.

Reply:
IMO we should then refresh images and move to up to date schema. But this is just nitpick, kiwi is just forced to run xslt script before each build.